KotlinMMP/shared/src/commonTest/kotlin/and/jemison/kotlinmmp/shared/MoodServiceTest.kt

```diff
@@ -0,0 +1,21 @@
+package and.jemison.kotlinmmp.shared
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class MoodServiceTest {
+
+    @Test
+    fun getMoodFromId0() {
+        val mood = MoodService().getMood(0)
```

Comment:
Not sure how I feel about the names of these tests. Would it be better to define an interface and create classes for each mood type, then loop through in one test? Not sure how many moods we'll end up with